cmd/utils/vmss.go

```diff
@@ -125,6 +134,12 @@ func RunCommand(
 
 	client := armcompute.NewVirtualMachineScaleSetVMsClient(vm.SubscriptionID, cred, nil)
 
+	// By default, the Azure API limits the output to the last 4,096 bytes. See
+	// https://learn.microsoft.com/en-us/azure/virtual-machines/linux/run-command#restrictions.
+	if outputTruncate == OutputTruncateTail {
+		*command = fmt.Sprintf("%s | head -c %d", *command, BytesLimit)
```

Comment:
Do we know how does it behave together with `stderr` like is this `4096` bytes limit for entire output or just `stdout`?